libs/shared/ui-components/src/Footer/Footer.tsx

```diff
@@ -9,7 +9,7 @@ export const Footer: FC<TFooterProps> = ({
   policyAndConditions,
   copyright,
 }) => (
-  <footer className="flex justify-center items-center text-black bg-white xl:text-white xl:bg-black">
+  <footer className="flex justify-center items-center text-black bg-white xl:text-[#efefef] xl:bg-black">
```

Comment:
Pure white text on pure black creates a halo and strains the eyes. This is just a notch darker than pure white, but still meets WCAG AA and AAA for normal text. This changes the white text in the footer for both Labs and Consulting. Wen the footer is wide, it has a black background and white text.